src/fondant/filesystem.py

```diff
@@ -0,0 +1,40 @@
+"""This module defines common filesystem functionalities."""
+import logging
+
+import fsspec
+
+logger = logging.getLogger(__name__)
+
+FSSPEC_SCHEME_DICT = {
+    "file": "file",
+    "s3": "s3",
+    "gs": "gcs",
+    "abfs": "abfs",
+}
```

Comment:
I don't think we need this mapping. `fsspec.filesystem()` works with the protocol directly.

Reply:
you're right, adjusted it